frontend/src/lib/forms/FormFieldError.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+  export let error: string | string[] | undefined;
+  export let id: string;
+</script>
+
+{#if error}
+  <label for={id} class="label">
+    <span class="label-text-alt text-error">{typeof error === 'string' ? error : error.join(', ')}</span>
```

Comment:
if error is undefined then this code will try to call join on it.

Reply:
nevermind I see that this is wrapped in `if error` so it won't be undefined here.